aspnetcore/docfx.json

```diff
@@ -60,6 +60,7 @@
       "ms.technology": {
         "azure/**/**.md": "aspnetcore-azure",
         "client-side/**/**.md": "aspnetcore-clientside",
+        "client-side/spa/blazor/**.md": "aspnetcore-blazor",
```

Comment:
I'll get the metadata whitelisting request submitted for this one.

Reply:
You did .... it's `aspnetcore-razorcomponents` that needs whitelisting.

Reply:
Okay. It's good to go now.

Reply:
https://exmail.aquent.com/owa/redir.aspx?C=0-DBeAri4cutwNn2onxhOBlRx8bk_fraTdb7xvxiXOBhtA5fNpHWCA..&URL=https%3a%2f%2fceapex.visualstudio.com%2fEngineering%2f_workitems%2fedit%2f64140

Reply:
I'll email you the link.